README.md

```diff
@@ -1,2 +1,367 @@
+
+You need to do it every time before you work with the cluster (run `kubectl` or `kops` commands, validate cluster `kops validate cluster`, etc.) after it has been created.
+
+<br/>
+
+
+### Provision `vpc` from `backing-services` with Terraform
+
+__NOTE:__ We provision `backing-services` in two phases because:
+
+* `aurora-postgres` and `elasticache-redis` depend on `vpc-peering` (they use `kops` Security Group to allow `kops` applications to connect)
+* `vpc-peering` depends on `vpc` and `kops` (it creates a peering connection between the two networks)
+
+To break the circular dependencies, we provision `kops`, then `vpc` (from `backing-services`), then `vpc-peering`,
+and finally the rest of `backing-services` (`aurora-postgres` and `elasticache-redis`).
+
+__NOTE:__ We use `chamber` to first populate the environment with the secrets from the specified service (`backing-services`)
+and then execute the given commands (`terraform plan` and `terraform apply`)
+
+
+```bash
+cd /conf/backing-services
+init-terraform
+chamber exec backing-services -- terraform plan -target=module.identity -target=module.vpc -target=module.subnets
+chamber exec backing-services -- terraform apply -target=module.identity -target=module.vpc -target=module.subnets
+```
+
+
+### Provision `vpc-peering` from `kops-aws-platform` with Terraform
+
+```bash
+cd /conf/kops-aws-platform
+init-terraform
+terraform plan -target=module.identity -target=data.aws_vpc.backing_services_vpc -target=module.kops_vpc_peering
+terraform apply -target=module.identity -target=data.aws_vpc.backing_services_vpc -target=module.kops_vpc_peering
+```
+
+
+### Provision the rest of `kops-aws-platform` with Terraform
+
+```bash
+cd /conf/kops-aws-platform
+terraform plan
+terraform apply
+```
+
+
+### Provision the rest of `backing-services` with Terraform
+
+__NOTE:__ We use `chamber` to first populate the environment with the secrets from the specified service (`backing-services`)
+and then execute the given commands (`terraform plan` and `terraform apply`)
+
+```bash
+cd /conf/backing-services
+chamber exec backing-services -- terraform plan
+chamber exec backing-services -- terraform apply
+```
+
+
+
+### Provision Kubernetes resources
+
+We use [helmfile](https://github.com/roboll/helmfile) to deploy [Helm](https://helm.sh/) [charts](https://github.com/kubernetes/charts) to provision Kubernetes resources.
+
+`helmfile.yaml` is located in the `/conf/kops` directory in `geodesic` container (see [helmfile.yaml](https://github.com/cloudposse/geodesic/blob/master/conf/kops/helmfile.yaml)).
+
+Change the current directory to `kops`
+
+```bash
+cd /conf/kops
+```
+
+Deploy the Helm charts
+
+__NOTE:__ We use `chamber` to first populate the environment with the secrets from the `kops` service and then execute the given command (`helmfile sync`)
+
+``` bash
+kops export kubecfg $KOPS_CLUSTER_NAME
+chamber exec kops -- helmfile sync
```

Comment:
FYI
I had to install RBAC roles and stuff first

file: role-based-access-control-config.yaml
```
apiVersion: v1
kind: ServiceAccount
metadata:
  name: tiller
  namespace: kube-system
---
apiVersion: rbac.authorization.k8s.io/v1beta1
kind: ClusterRoleBinding
metadata:
  name: tiller
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: cluster-admin
subjects:
  - kind: ServiceAccount
    name: tiller
    namespace: kube-system
```

then create the roles and bindings
`chamber exec kops -- kubectl create -f 'role-based-access-control-config.yaml'  --validate=false`

then stop the tiller deployment, 
`chamber exec kops -- kubectl -n "kube-system" delete deployment tiller-deploy`

then restart tiller with
`chamber exec kops -- helm init --service-account tiller`

Reply:
@tamsky 
thanks for this.